tools/bqtag/tests/test_bqtag.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+import json
+from bqtag import BQTableView
+
+"""
+- Update the BQ_PROJECT, CATALOG_PROJECT, BQ_DATASET, LOCATION and TAXONOMY_DISPLAY_NAME with relevant values.
+- Key file used is credentials.json. Please add the correct path or copy credentials.json to the tests folder. 
+- SA should have relevant permissions to read Data Catalog Tags otherwise table creation will result in error.
+"""
```

Comment:
The docstring should be at the top, and before any import

Reply:
changed to:

```
"""
- Update the BQ_PROJECT, CATALOG_PROJECT, BQ_DATASET, LOCATION and TAXONOMY_DISPLAY_NAME with relevant values.
- Key file used is credentials.json. Please add the correct path or copy credentials.json to the tests folder. 
- SA should have relevant permissions to read Data Catalog Tags otherwise table creation will result in error.
"""

import unittest
import json
from bqtag import BQTableView

BQ_PROJECT = None #Update this with BQ Project Value
```